File: 프로그래머스/JS/68935삼진법뒤집기.js
function solution(n) {
    var answer = [];
    let num = 0;
    
    for(let i=n; i>0; i=parseInt(i/3)){
        answer.push(i%3);
    }
    for(let i=0; i<answer.length; i++){
        num += answer[i] * Math.pow(3, answer.length-i-1);
    }
    return num;
}

File: 프로그래머스/C++/68935삼진법뒤집기.cpp
#include <string>
#include <vector>
#include <cmath>
using namespace std;

int solution(int n) {
    int answer = 0;
    vector<int> v;
    for(int i=n; i>0; i=i/3){
        v.push_back(i%3);
    }

    for(int i=0; i<v.size(); i++){
        answer += v[i]* pow(3,v.size()-i-1);
    }
    return answer;
}
